Data_Portal_UG.yml

```diff
@@ -14,13 +14,11 @@ pages:
   - Exploration: Data_Portal/Users_Guide/Exploration.md
   - Repository: Data_Portal/Users_Guide/Repository.md
   - Genes and Mutations: Data_Portal/Users_Guide/Genes_and_Mutations.md
-  - Custom Set Analysis: Data_Portal/Users_Guide/Custom_Set_Analysis.md  
```

Comment:
Is Custom Set Analysis supposed to be removed for legacy archive docs updates? Or is this to get ahead of the docs for GDC v2?

Reply:
This is meant to be removed.  Going to remove file too.